src/definitions/behaviors/form.js

```diff
@@ -136,10 +136,7 @@
                 },
 
                 attachEvents: function (selector, action) {
-                    if (!action) {
```

Comment:
We have explicitely added this in the previous PRs...now you remove in again? Can we please find a "final standard" :)